test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -24,6 +24,10 @@ public abstract partial class BaseTestBaseline : BaseTestClass
     {
         public const int DigitsOfPrecision = 7;
 
+        public static bool NotNetCore30 { get; } = AppDomain.CurrentDomain.GetData("FX_PRODUCT_VERSION") == null ? true : false;
```

Comment:
This isn't written correctly. When `netcoreapp3.1` or `4.0` comes along, this code will break.

Reply:
why the code will break ? 
```AppDomain.CurrentDomain.GetData("FX_PRODUCT_VERSION") == null``` wont be false for any versions greater than or equal to 3.0 ?

Reply:
Right, but it also won't give the right answer for `NotNetCore30`.

Reply:
how about ```LessThanNetCore30OrNotNetCore ``` ?

Reply:
That seems like a more accurate name.